services/web/server/src/simcore_service_webserver/long_running_tasks/plugin.py

```diff
@@ -31,7 +30,9 @@ async def _test_task_context_decorator(
     ) -> web.StreamResponse:
         """this task context callback tries to get the user_id from the query if available"""
         req_ctx = AuthenticatedRequestContext.model_validate(request)
-        request[RQT_LONG_RUNNING_TASKS_CONTEXT_KEY] = jsonable_encoder(req_ctx)
+        request[LONG_RUNNING_TASKS_CONTEXT_REQKEY] = req_ctx.model_dump(
+            mode="json", by_alias=True
```

Comment:
Q: are you sure this has no sideeffect?

Reply:
actually it had when i did not do it :-)
`jsonable_encoder` defaults to `by_alias=True` but pydantic v2 `model_dump` defaults to `by_alias=False`